src/features/MovieList/MovieTile/styled.js

```diff
@@ -66,27 +68,31 @@ export const Year = styled.p`
     }
 `;
 
-export const TagsContainer = styled.div`
+export const TagsContainer = styled.ul`
     margin-top: 8px;
     display: flex;
     flex-wrap: wrap;
     gap: 8px;
+    padding: 0;
+    list-style: none;
+    margin-bottom: 0;
 `;
 
-export const Tag = styled.p`
+export const Tag = styled.li`
     margin: 0px;
     padding: 8px 16px;
     background-color: ${({ theme }) => theme.color.gray};
     border-radius: 5px;
     font-size: 14px;
+    
     @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
         padding: 4px 8px;
         font-size: 10px;
     }
 `;
 
 export const RatingContainer = styled.div`
-    margin-top: 39px;
+    margin-top: 14px;
     display: flex;
     align-items: center;
     align-content: flex-end;
```

Comment:
align-content does not do anything here.

Reply:
align-content change everythink look again delete this and paste its very important line of code, 
if it wasn't there the rating will go up, as we know we want rating stick to bottom

Reply:
Check it once again, in my opinion it does not do anything. You probably thought of `justify-content: space-between;` property in Wrapper component, this is RatingContainer.

![no-align-content](https://user-images.githubusercontent.com/98344543/176688558-55b26e24-2881-4c36-a57b-bf00c56b32b1.jpg)
![align-content](https://user-images.githubusercontent.com/98344543/176688566-7ee0987b-7a66-499e-83e0-1c2a6eb45d2b.JPG)

Also align-content has no effect on single line flex containers. In the time i made this comment this had default flex-wrap which is no-wrap - that meant it had to be single line. Here is an [align-content documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/align-content). Besides do you think `flex-wrap: wrap` you added is needed here?

Reply:
Okay, i pulled your latest changes, I guess we need it now.

Reply:
if you want space between we need an extra container, better is align content I show you on daily, maybe our browsers work differently because on mine it looks different

Reply:
> if you want space between we need an extra container, better is align content I show you on daily, maybe our browsers work differently because on mine it looks different

No, its just at the time i made the comment i haven't got your latest changes. align-content is ok with your current solution.